Source/FieldSolver/SpectralSolver/SpectralKSpace.H

```diff
@@ -15,8 +15,8 @@
 // `KVectorComponent` and `SpectralShiftFactor` hold one 1D array
 // ("ManagedVector") for each box ("LayoutData"). The arrays are
 // only allocated if the corresponding box is owned by the local MPI rank.
-using KVectorComponent = amrex::LayoutData<
-                           amrex::Gpu::ManagedVector<amrex::Real> >;
+using RealKVector = amrex::Gpu::ManagedVector<amrex::Real>;
```

Comment:
not ideal to define those here at public scope. Please either:
- move into a struct/class `public` section and/or
- add a namespace

Reply:
This should probably be a separate PR since it will also effect the Cartesian spectral solver.